rust-runtime/aws-smithy-runtime/src/client/orchestrator/auth.rs

```diff
@@ -151,6 +151,10 @@ pub(super) async fn resolve_identity(
                         } else {
                             IdentityCache::no_cache()
                         };
+                        // Apply properties from the selected auth scheme option
+                        if let Some(properties) = options[i].properties() {
```

Comment:
why doesn't `auth_scheme_option.properties()` work? (why is enumerate necessary I guess).